pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -1512,9 +1507,13 @@ assertNoAdditions {
   };
 
   indent-tools-nvim = super.indent-tools-nvim.overrideAttrs {
+    checkInputs = [
+      # nvim-treesitter-textobjects dep
+      self.nvim-treesitter
+    ];
```

Comment:
question: why is this needed here? I can't see any tests in indent-tools.nvim (unless I'm looking at the wrong plugin).